charts/trento-server/charts/trento-mcp-server/values.yaml

```diff
@@ -0,0 +1,255 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+---
+## @section Global parameters
+##
+## @param global.trentoMcpServer.servicePort Global service port for Trento MCP Server
+global:
+  trentoMcpServer:
+    servicePort: ""
+
+## @section Common parameters
+##
+## @param image.repository Image repository
+## @param image.tag Image tag
+## @param image.pullPolicy Image pull policy
+image:
+  repository: ghcr.io/trento-project/trento-mcp-server
+  tag: latest
+  pullPolicy: Always
+
+## @param replicaCount Number of replicas to deploy
+replicaCount: 1
+
+## @param containerPorts.http Port for HTTP traffic
+## @param containerPorts.health Port for health check
+containerPorts:
+  http: 5000
+  health: 8080
+
+## @section MCP Server configuration
+##
+## @param mcpServer.args Array of arguments to pass to the MCP server container, overrides other settings
+## @param mcpServer.enableHealthCheck Enable health check endpoint
+## @param mcpServer.transport Transport protocol for the server
+## @param mcpServer.oasPath List of paths to OpenAPI specification files
+## @param mcpServer.trentoURL URL of the Trento server
+## @param mcpServer.headerName Name of the header for the API key
+## @param mcpServer.tagFilter List of tags to filter
+## @param mcpServer.verbosity Log level verbosity
+## @param mcpServer.insecureTLS Disable TLS certificate verification
+mcpServer:
+  args: []
+  enableHealthCheck: true
+  transport: streamable
+  oasPath:
+    - https://www.trento-project.io/web/swaggerui/openapi.json
+    - https://www.trento-project.io/wanda/swaggerui/openapi.json
+  trentoURL: https://demo.trento-project.io
+  headerName: X-TRENTO-MCP-APIKEY
+  tagFilter:
+    - MCP
+  verbosity: info
+  insecureTLS: false
+
+## @param config.enabled Enable using a custom configuration file
+## @param config.fileName Name of the configuration file
+## @param config.mountPath Path to mount the configuration file
+## @param config.content Content of the configuration file
+config:
+  enabled: false
+  fileName: trento-mcp-server.config.yaml
+  mountPath: /etc/trento
+  content: ""
```

Comment:
this is to mimic the behavior of the config file used in the RPM, so we can re-use the same instructions. However, we might not actually need it (it's optional for now), I'm open to either leave it or remove it.